src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTestsUsingCertWithPrivateKey.cs

```diff
@@ -67,5 +67,50 @@ public static void DecryptMultipleRecipients()
             ContentInfo contentInfo = ecms.ContentInfo;
             Assert.Equal<byte>(content, contentInfo.Content);
         }
+
+        [Fact]
```

Comment:
This change is unrelated but I noticed this test does not run because (Xunit doesn't run static tests in abstract class)